pkgs/by-name/ne/neohtop/package.nix

```
@@ -0,0 +1,78 @@
+{
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  jq,
+  stdenv,
+  fetchNpmDeps,
+  zsh,
+  fetchFromGitHub,
+  npmHooks,
+  lib,
+  openssl,
+}:
+
+let
+  tauri = cargo-tauri;
+
+in
+rustPlatform.buildRustPackage rec {
+  name = "neohtop";
+  pname = name;
```

`name` is formed from `pname` and `version`
```suggestion
  pname = "neohtop";
```